static/app/components/assigneeSelectorDropdown.tsx

```diff
@@ -87,6 +87,10 @@ export interface AssigneeSelectorDropdownProps {
    * Optional list of suggested owners of the group
    */
   owners?: Omit<SuggestedAssignee, 'assignee'>[];
+  /**
+   * Additional styles to apply to the dropdown
+   */
+  style?: React.CSSProperties;
```

Comment:
I don't think you want to add this prop - `style` should only be used for dynamic styles (they override all CSS which causes issues for styling down the road). Can you add a `className` instead?